cmd/openqa-revtui/tui.go

```diff
@@ -436,7 +436,7 @@ func (tui *TUI) buildHeader(width int) []string {
 	lines := make([]string, 0)
 	if tui.header != "" {
 		lines = append(lines, tui.header)
-		lines = append(lines, "q:Quit   r:Refresh   h:Hide/Show jobs   m:Toggle RabbitMQ tracker   s:Switch sorting    Arrows:Move up/down")
+		lines = append(lines, "q:Quit   r:Refresh   h:Hide/Show jobs o: Open in browser  m:Toggle RabbitMQ tracker   s:Switch sorting    Arrows:Move up/down")
```

Comment:
```suggestion
		lines = append(lines, "q:Quit    r:Refresh    h:Hide/Show jobs    o:Open links    m:Toggle RabbitMQ tracker   s:Switch sorting    Arrows:Move up/down")
```

I think "Open links" is more descriptive than "Open in browser", and let's keep the spacing between the elements consistent

Reply:
done